### PR TITLE
Update unenroll-your-device-from-intune-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed the remove option (company portal ) as it is no longer available

It has only Rename, Reset and Reset PIN options

Issue#https://github.com/MicrosoftDocs/memdocs/issues/2977

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger19** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/user-help/unenroll-your-device-from-intune-windows.md [hunk 1]
```diff
@@ -78,32 +78,6 @@ After you unenroll a device running Windows 8.1 RT:
 ## Remove Windows 10/11 devices  
 This section describes how to remove a Windows 10/11 device from Intune. If one set of instructions doesn't work for you, try the next set.
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/unenroll-your-device-from-intune-windows.md:79]:
```suggestion
This section describes how to remove a Windows 10/11 device from Intune.   
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger7** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and signoff this Pr. Thanks

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**